### PR TITLE
Keep the findBestExp method exception consistent between HepPlanner and VolcanoPlanner

### PR DESCRIPTION
**Pull request by ChengJie1053**

### CODE REVIEW

**Review by mihaibudiu — COMMENTED** [review]
I personally prefer to quote the identifiers in question in such messages:
'root' must not be null

**Comment by ChengJie1053** [comment]
> I personally prefer to quote the identifiers in question in such messages: 'root' must not be null

Ok, thank you. I'll change the code

**Review by mihaibudiu — APPROVED** [review]

**Comment by caicancai** [comment]
@ChengJie1053 Please git squash

**Comment by ChengJie1053** [comment]
> @ChengJie1053 Please git squash

Ok, thank you

**Review by caicancai — APPROVED** [review]

File: core/src/main/java/org/apache/calcite/plan/hep/HepPlanner.java [hunk 1]
```diff
@@ -186,14 +186,14 @@ public HepPlanner(
   }
 
   @Override public RelNode findBestExp() {
-    requireNonNull(root, "root");
+    requireNonNull(root, "'root' must not be null");
```
Comment (NobiGo) [inline core/src/main/java/org/apache/calcite/plan/hep/HepPlanner.java:189]: What is the error message before this PR? I want to know `must not be null` it is repetitive with `requireNonNull`? The other places in this project all use it like `requireNonNull(root, 'root');`

Comment (ChengJie1053) [reply core/src/main/java/org/apache/calcite/plan/hep/HepPlanner.java:189]: Ok, thanks for reviewing the code for me. I was initially trying to get the same error message from findBestExp for HepPlanner and VolcanoPlanner
image

Comment (ChengJie1053) [reply core/src/main/java/org/apache/calcite/plan/hep/HepPlanner.java:189]: Ok, thanks for reviewing the code for me

**Comment by NobiGo** [comment]
@ChengJie1053  Hi, How about we change the commit info to `Keep the findBestExp method exception consistent between HepPlanner and VolcanoPlanner` ?

**Comment by ChengJie1053** [comment]
> @ChengJie1053 Hi, How about we change the commit info to `Keep the findBestExp method exception consistent between HepPlanner and VolcanoPlanner` ?

Ok, thanks for reviewing the code for me

**Comment by NobiGo** [comment]
@ChengJie1053 Please squash the commits. If the CI pass, I will merge it.

**Comment by NobiGo** [comment]
@ChengJie1053 Thanks for the contribution.

**Pull request merged**